sdk/src/main/java/io/opentelemetry/sdk/trace/SpanBuilderSdk.java

```diff
@@ -137,6 +138,9 @@
 
   @Override
   public Span.Builder setAttribute(String key, String value) {
+    if (StringUtils.isNullOrBlank(value)) {
```

Comment:
Do we want to count this as dropped or we simply ignore?

Reply:
it's definitely a different kind of "dropping" than the "too many attributes" kind. I think ignoring it is fine..it's not really an attribute at all without a value.

Reply:
Fine with this for the moment, may be unexpected for users especially for empty.